www/js/model.js

```diff
@@ -174,7 +174,7 @@ define([
 			},
 		};
 
-		var obj.authored = lang.mixin({
+		obj["authored"] = lang.mixin({
```

Comment:
@riteshsamala shouldnt this be obj.authored as we had discussed. Only the var had to be removed. Was it still causing some issues?